frontend/src/features/sketches/SketchModal.tsx

```diff
@@ -0,0 +1,10 @@
+import { Modal } from "~/shared/components/Modal";
+type Props = {
+    selectedSketchUrl: string;
+    setSelectedSketchUrl: (url: string | null) => void;
+};
+
+export const SketchModal = ({ selectedSketchUrl, setSelectedSketchUrl }: Props) =>
+  <Modal isOpen={true} onClose={() => setSelectedSketchUrl(null)}>
+    <img src={selectedSketchUrl} alt={selectedSketchUrl} />
+  </Modal>;
```

Comment:
[nits]
中でreturnしてあげる方が一般的な気がするから統一しておきたいかも！
```suggestion
export const SketchModal = ({ selectedSketchUrl, setSelectedSketchUrl }: Props) => {
  return (
    <Modal isOpen={true} onClose={() => setSelectedSketchUrl(null)}>
      <img src={selectedSketchUrl} alt={selectedSketchUrl} />
    </Modal>
  );
};
```